docs/configuration.asciidoc

```diff
@@ -239,5 +239,22 @@ By default the agent can only send (up to) `20` transactions every `30000` milli
 A number between `0.0` and `1.0` that specifies the sample rate of transactions, by default all transactions are sampled.
 
 
+[float]
+[[central-config]]
+==== `centralConfig`
+
+* *Type:* Boolean
+* *Default:* `false`
+
+This option activates APM Agent Configuration via Kibana.
+If set to `true`, the agent will fetch the central configuration via the APM Server on the initialization phase.
+The agent caches the central configuration in the `sessionStorage` and it does not fetch them again until
+the session is closed and/or `sessionStorage` is cleared, in most cases this means when the tab/window of the page is closed.
+
+NOTE: Currently, only <<transaction-sample-rate, transaction sample rate>> can be configured via Kibana.
+
+NOTE: This feature requires APM Server v7.5 or later and that the APM Server is configured with `kibana.enabled: true`.
+More information is available in {kibana-ref}/agent-configuration.html[APM Agent configuration].
```

Comment:
I'm in the process of updating the attribute we use to link to the APM app docs (because they're gonna be moving). Can you use `apm-app-ref` instead please?

```suggestion
More information is available in {apm-app-ref}/agent-configuration.html[APM Agent configuration].
```